README.md

```diff
@@ -32,19 +32,27 @@ Single Repository:
 
 ```sh
 cd /path/to/my/repository
-git config --local commit.gpgsign true  # Sign all commits
-git config --local tag.gpgsign true  # Sign all tags
 git config --local gpg.x509.program gitsign  # Use gitsign for signing
 git config --local gpg.format x509  # gitsign expects x509 args
+
+# Warning: Setting this will make git commit/tag reliant on internet.
+# Alternatively, don't use these settings and add the -S flag instead.
+git config --local commit.gpgsign true  # Sign all commits
+git config --local tag.gpgsign true  # Sign all tags
+
 ```
 
 All respositories:
 
 ```sh
-git config --global commit.gpgsign true  # Sign all commits
-git config --global tag.gpgsign true  # Sign all tags
 git config --global gpg.x509.program gitsign  # Use gitsign for signing
 git config --global gpg.format x509  # gitsign expects x509 args
+
+
+# Warning: Setting this will make git commit/tag reliant on internet.
+# Alternatively, don't use these settings and add the -S flag instead.
```

Comment:
The `-S` flag for commits will still rely on the internet, no? I wonder if this message belongs elsewhere more prominently for gitsign use as a whole?

Reply:
Yeah, explaining that every signed commit now reaches out to the internet might be worth mentioning somewhere. I had setup this setting and then couldn't figure out why commits were happening so slowly.

Reply:
perhaps adding some text in the signing commits and signing tags sections makes sense? https://github.com/sigstore/gitsign/blob/main/README.md?plain=1#L115

Reply:
will be good to add some clarification in this section as well